packages/core/src/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -42,11 +42,17 @@ ReactDOM.render(
       ]}
       label="Select from the options below:"
       onChange={selectedItem => console.log(selectedItem)}
+      onInputValueChange={inputVal =>
+        console.log('[Autocomplete]: ' + inputVal)
+      }
     >
       <TextField
         hint="Type c then use ARROW keys to change options, ENTER key to make a selection, ESC to dismiss."
         label="Labeled list"
         name="Downshift_autocomplete"
+        onBlur={() => console.log('[TextField]: Blur event')}
+        onChange={e => console.log(e)}
+        onKeyDown={() => console.log('[TextField]: Keydown event')}
```

Comment:
I think all of these placeholder event listeners are nice for debugging, but as a developer implementing the design system, I would be confused about which one I actually need to include. Can we reduce this to just one event handler that someone would need to be aware of in order to make an API call to get an updated list of items? Is it `onInputValueChange` that would be best to use?

Reply:
I'm split on how much to include, and how much to leave out. The Autocomplete `onInputValueChange` prop is probably the highest level, easiest way to make an API call. I can remove these extra event listeners from the examples, but feel they're worth leaving in the cloned component. Do you both concur?

Reply:
yea, my comment is only directed towards the example code we put in the docs. I don't think our intent is for a developer to add all of these event handlers, so wouldn't want to advocate doing that 😛